docs/Collaboration guidelines.md

```diff
@@ -0,0 +1,51 @@
+Collaboration Guidelines
+===
+
+Be polite to each other
+---
+
+No one wants to wake up to a rude reply. Try to behave the best way you can, and show understanding when others are not. 
+
+When things escalate, try to use this document as a guideline on how to de-escalate them.
+
+Think of the project first
```

Comment:
I like this point, but think it could be a little clearer. "Put the project before your personal feelings"